spec/scripts/integer_2_spec.rb

```diff
@@ -17,7 +17,7 @@
 end
 
 describe "integer_birth_year.rb" do
-  it "should output 'Wow, you were born in 1940. You're old!' if the input is 80.", points: 1 do
+  it "should output 'Wow, you were born in 1943. You're old!' if the input is 80.", points: 1 do
```

Comment:
This definitely works for this year. Next year, the issue will pop up again no doubt (unless someone remembers to update these kinds of assignments beforehand).

It'd be really nice if the spec title could be dynamic and stay up to date even when the year changes (#6).

I imagine this could be possible using String interpolation:
```suggestion
  it "should output 'Wow, you were born in #{Date.today.year - 80}. You're old!' if the input is 80.", points: 1 do
```

What do you think?